### ISSUE TITLE
Circ / Flash weird api issue

### ISSUE THREAD

**Issue by MBryde** [ubahnverleih/WoBike#133]
Hi. When requesting vehicles on almost any other zone than the predefined one in [Flash.md](https://github.com/ubahnverleih/WoBike/blob/master/Flash.md) you get a response back with a different location that do not match the lat/lng you put in.

Do anyone know if their is some sort of formula you need to apply the lat/lng to get the correct cities?

**Comment by coon42** [comment]
My theory is, that this API is deprecated and new countries which where added after they renamed themself from flash to circ have not been added.

**Comment by coon42** [comment]
Yep, some countries like germany and austria are not supported by their old API.
See [#95](https://github.com/ubahnverleih/WoBike/issues/95#issuecomment-509297899) on how to use their new API.
